src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -947,6 +954,14 @@ export class PatientViewPageStore {
     @action clearErrors() {
         this.ajaxErrors = [];
     }
+    
+    @autobind
+    @action toggleModal(name?:string|undefined) {
```

Comment:
"toggle" strongly implies that you are just turning something on or off.  I would